README.md

```diff
@@ -1,24 +1,29 @@
+- **Defense in Depth**: The system is designed with multiple layers of security. Restrictions on times when transactions can be queued limit activity to certain days and times of the week. The timelock prevents transactions from being executed immediately. Dark spells can be used to create custom recovery logic for a multisig. The dark spell could either completely rotate the multisig signer set, or contain more complex logic to recover funds in case of lost keys.
+
 # Safe Transaction Time Restriction
 
 This contract allows a Gnosis Safe the ability to restrict the time window in which a transaction can be executed. It is useful, for example, to prevent a user from withdrawing funds from a contract during a certain time window. For DAO's it can set operating hours for a treasury multisig, and for a user, it can prevent a user from withdrawing funds from a personal multisig during certain hours.
 
 ## Architecture
+![](Architecture.png)
 
-This contract is a singleton contract. Once deployed, to be able to use this contract you should call the `addTimeRange` function to add a time range in which transactions are allowed.
-
-```solidity
-    /// @param dayOfWeek day of the week to allow transactions
-    /// - valid range [1, 7]
-    /// @param startHour start hour of the allowed time range
-    /// - valid range: [0, 23]
-    /// @param endHour end hour of the allowed time range
-    /// - valid range: [0, 23]
-    function addTimeRange(
-        uint8 dayOfWeek,
-        uint8 startHour,
-        uint8 endHour
-    ) external;
-```
+Each system component works together to ensure a user cannot be coerced into signing actions they do not want to take. 
+
+- **Time Restriction Contract**: This contract restricts the time window in which a transaction can be executed. Additionally, it stops owners from being rotated out of the multisig (except by modules), stops additional modules from being added or removed during a transaction, and prevents the guard from being disabled by a transaction. With these checks in place, it is impossible to remove the guard, rotate signers, or add new modules to the multisig without a timelocked transaction passing.
+- **Timelock Contract**: This contract is a delay mechanism for transactions. It requires a delay before a transaction can be executed. This delay can be set to any time period between 1 and 30 days. This delay prevents an attacker from immediately executing a transaction after coercing a user into signing a transaction. The timelock can whitelist contract calls and calldata, this allows safe signers to execute interactions with other smart contracts as long as the contract, function signature, and certain parts of the calldata are whitelisted.
```

Comment:
Just to clarify, whitelisted calls under timelock will still be subject to the Time Restriction guard, correct?